content/en/docs/contribute/localization.md

```diff
@@ -205,14 +205,21 @@ To ensure accuracy in grammar and meaning, members of your localization team sho
 
 ### Source files
 
-Localizations must be based on the English files from the most recent release, {{< latest-version >}}.
+Localizations must be based on the English files from a specific release targeted by the localization team.
+Each localization team can decide which release to target which is referred to as the _target version_ below.
 
-To find source files for the most recent release:
+To find source files for your target version:
 
 1. Navigate to the Kubernetes website repository at https://github.com/kubernetes/website.
-2. Select the `release-1.X` branch for the most recent version.
+2. Select branch for your target version from followings.
 
-The latest version is {{< latest-version >}}, so the most recent release branch is [`{{< release-branch >}}`](https://github.com/kubernetes/website/tree/{{< release-branch >}}).
+Target version | Branch
+-----|-----
+Next version | [`dev-{{< skew nextMinorVersion >}}`](https://github.com/kubernetes/website/tree/dev-{{< skew nextMinorVersion >}})
+Latest version | [`master`](https://github.com/kubernetes/website/tree/master)
+Previous version | `release-*.**`
+
+`master` branch holds content for the current release `{{< latest-version >}}`. And release team will create `{{< release-branch >}}` branch just before the next release, v{{< skew nextMinorVersion >}}.
```

Comment:
```suggestion
The `master` branch holds content for the current release `{{< latest-version >}}`. The release team will create `{{< release-branch >}}` branch shortly before the next release: v{{< skew nextMinorVersion >}}.
```

Maybe link to https://github.com/kubernetes/sig-release/tree/master/release-team/#kubernetes-release-team-roles ?